core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -108,14 +108,14 @@ class ConsoleConsumerTest extends JUnitSuite {
     val config = new ConsoleConsumer.ConsumerConfig(args)
 
     //Then
-    assertTrue(config.useNewConsumer)
+    assertFalse(config.useOldConsumer)
```

Comment:
OK. Not sure what happened and why GitHub repeated my question several times. :) I deleted the other ones.